sdk/src/client/node_api/core/routes.rs

```diff
@@ -333,6 +334,17 @@ impl ClientInner {
         self.get_request(path, None, false, false).await
     }
 
+    /// Get all full UTXO changes of a given slot by slot commitment ID.
+    /// GET /api/core/v3/commitments/{commitmentId}/utxo-changes/full
+    pub async fn get_utxo_changes_full_by_slot_commitment_id(
```

Comment:
I would suggest the name `get_full_utxo_changes_by_slot_commitment_id` tbh, but ignore if you disagree.

Reply:
I would keep this way so it's more consistent with the URL